provision/provision-site.sh

```diff
@@ -111,7 +111,7 @@ if [[ true == $SKIP_PROVISIONING ]]; then
   return
 fi
 
-echo -e "${BLUE}Running provisioner for site ${SITE}${CRESET}"
+echo -e "${GREEN}Running provisioner for site ${SITE}${CRESET}"
```

Comment:
Shouldn't we standarize colors like this?

Green: Success
Red: Error
Yellow: Warning
Blue: Info

That's the reason I used blue.